torch/_inductor/codegen/cuda/cuda_kernel.py

```diff
@@ -0,0 +1,272 @@
+from typing import List, Optional
+
+import sympy
+
+from ...autotune_process import CUDABenchmarkRequest
+from ...ir import Callable, CUDATemplateBuffer, IRNode, Layout, TensorBox
+from ...select_algorithm import ChoiceCaller
+from ...utils import sympy_product
+from ...virtualized import V
+
+from ..common import IndentedBuffer, Kernel, OpOverrides
+from ..cpp import CppPrinter, DTYPE_TO_CPP
+
+
+cexpr = CppPrinter().doprint
+
+
+def _normalize_idx(index: int, total_length: int) -> int:
+    return index if index >= 0 else index + total_length
+
+
+class CUDAKernel(Kernel):
+    """
+    Kernels defined by the CUDA language.
+    """
+    overrides = OpOverrides
```

Comment:
Which consequences does this line have? It's a bit hard to read out of the inductor codegen, how exactly these overrides fields are used.